### ISSUE TITLE
"ImportError: No module named opencage.geocoder"

### ISSUE THREAD

**Issue by shihas1991** [OpenCageData/python-opencage-geocoder#8]
I just go through the sign up process and generate api key. For importing the api from python https://github.com/opencagedata/python-opencage-geocoder .I am getting an error "ImportError: No module named opencage.geocoder"

**Comment by sabarasaba** [comment]
I'm no python expert, but I managed to make it work by just cloning the repo:

`git clone https://github.com/jkbr/httpie.git`

Then just run the setup.py file from that directory:

`sudo python setup.py install`

Once we fix #5 it will only be a matter of doing `pip install opencage-geocoder`

**Comment by mtmail** [comment]
You're right. The module is now on https://pypi.python.org/pypi/opencage/1.1.0 and installable with pip.

**Comment by sabarasaba** [comment]
Awesome, thanks @mtmail ! :tada:

**Comment by shihas1991** [comment]
Thanks for the quick reply. Successfully installed the opencage with pip install opencage.But I stuck with an error
from opencage.geocoder import OpenCageGeocode
>>> key = '86f899726d896df4d1ac54e32109e56c'
>>> geocoder = OpenCageGeocode(key)
>>> results = geocoder.reverse_geocode(51.51024, -0.10303)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "opencage/geocoder.py", line 147, in reverse_geocode
    return self.geocode(_query_for_reverse_geocoding(lat, lng))
  File "opencage/geocoder.py", line 100, in geocode
    if six.PY2:
AttributeError: 'module' object has no attribute 'PY2'

**Comment by mtmail** [comment]
Thanks the new bug report.

This is fixed in version 1.1.1. You can install it by running

```
pip install -upgrade opencage
```

Explanation:

Basically your installed version of the package `six` is more than a year old. We require at least version 1.4. The `opencage` package now sets the dependency correct.

You can find the current version of the page `six` by running

```
python --version
python -c 'import six; print six.__file__; print six.__version__'
```

**Comment by shihas1991** [comment]
Wowww...Thanks ...issue solved
